Build.fsx

```diff
@@ -173,6 +173,7 @@ Target "PublishNuGetRelease" (fun _ -> publishPackagesToNuGet
                                             (getBuildParam "NuGetReleaseKey"))
 
 Target "CompleteBuild" (fun _ -> ())
+Target "PublishNuGetAll" (fun _ -> ()) 
```

Comment:
How are the changes in this file related to the commit message? Shouldn't this be part of another commit or even be committed separately?